packages/teams/src/utils/search-teams.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  searchGroups,
+  ISearchOptions,
+  ISearchResult,
+  IGroup,
+} from "@esri/arcgis-rest-portal";
+
+/**
+ * Search for teams
+ * @param {ISearchOptions} searchRequestOptions
+ * @returns {Promise<ISearchResult>}
+ */
+export function searchTeams(
```

Comment:
Talked w/ @tomwayson  and since we _know_ that this signature will change, and we don't want to have a breaking-change release every time we do so during development, we'd like you to rename this `_searchTeams(...)` to indicate this is still under development, and add `@private` to the JSDOC comment so it's not in the generated docs. Then, once we have this finalized, we can drop the `_`, and remove the `@private` and cut a "minor" bump release.